dags/top-downloads-from-sowflake.py

```diff
@@ -163,11 +233,37 @@ def post_top_downloads_to_slack(message: str) -> str:
         result = client.chat_postMessage(channel="topcharts", text=message)
         return result
 
-    top_downloads = get_top_downloads_from_snowflake()
```

Comment:
Not related to this line, but let's remove the redundant byte string on the slack message